doc/USER_MANUAL/06_fault_sources.tex

```diff
@@ -336,18 +394,60 @@ \section{Input Files}
 \end{lyxlist}
 
 \begin{lyxlist}{00.00.0000}
-\item [{\textbf{DATA/input\_file.txt}}] Heterogeneous stresses and friction
+\item [{\textbf{DATA/input\_file.txt}}] Heterogeneous stresses and friction for 
+linear slip weakening friction
 parameters are documented in page 10 of
 
 \begin{lyxlist}{00.00.0000}
 \item [{$\mathtt{EXAMPLES/tpv16/description/TPV16\_17\_Description\_v03.pdf}$}]~
 \end{lyxlist}
 
-To activate this feature, in fault\_solver\_dynamic.f90 set \textbf{TPV16}
-= .true. then re-compile the code.
+To activate this feature, in Par\_file\_faults name list \&\textbf{RUPTURE\_SWITCHES}, set TPV16=.TRUE..
 
 \end{lyxlist}
+\begin{lyxlist}{00.00.0000}
+\item [{\textbf{DATA/rsf\_hete\_input\_file.txt}}]  Heterogeneous stresses and friction input for 
+  rate and state friction. 
+To activate this feature, in Par\_file\_faults name list \&\textbf{RUPTURE\_SWITCHES}, set RSF\_HETE=.TRUE..
+
+  The format of  {\textbf{DATA/rsf\_hete\_input\_file.txt}} is as such, in the first line there are four integers
+
+  that are sequentially documenting the number of divisions along strike, number of divisions along 
+  
+  dip, cell size along strike , cell size along dip. Then the following N(N=NumberOfDivisionsAlongStrike
+  
+  * NumberOfDivisionsAlongDip) lines will document the stress and friction properties on the grid.
+  
+  There are a total of 13 columns.\\
+
+  Column1 = Along strike distance \\
```

Comment:
Yes, but I mean the columns, can be in a nested `enumerate` (or `lyxlist` if you prefer) instead of a bunch of `\\` line breaks.